doc/gimbal_lock.rst

```diff
@@ -25,7 +25,7 @@ from bottom to top:
 
 .. image:: images/spitfire_0.png
 
-Imagine we wanted to do a slight roll with the left wing tilting down
+Imagine we wanted to do a slight roll with the left-wing tilting down
```

Comment:
That doesn't look right to me - surely you wouldn't hyphenate left-arm; but maybe there's some grammar rule here I don't know.

Reply:
Sure, I would remove the hyphen. Thanks for the information.